src/Debug/Todo.purs

```diff
@@ -0,0 +1,18 @@
+module Debug.Todo
+( notImplementedYet
+, (???)
+, todo
+) where
+
+import Data.Symbol (class IsSymbol, SProxy, reflectSymbol)
+import Prim.TypeError (class Warn, Text)
+
+notImplementedYet :: forall a. Warn (Text "not implemented yet") => a
+notImplementedYet = crashWith "not implemented yet"
+
+infix 5 notImplementedYet as ???
+
+todo :: forall a s. IsSymbol s => Warn (Text s) => SProxy s -> a
```

Comment:
Shouldn't this be more like `"TODO: " + s`? Or do we not care?

Reply:
Can't find a typelevel `+` (except for the `Row` one).

Reply:
It's technically `Beside` but I saw one do it using `<>` via a type alias:
https://pursuit.purescript.org/packages/purescript-typelevel-eval/0.2.0/docs/Type.Eval#t:type%20(%3C%3E)

Reply:
Keep in mind that `Beside` will lay out the content in a quirky way if newlines are included in the type-level strings:
https://github.com/JordanMartinez/purescript-jordans-reference/blob/latestRelease/21-Hello-World/04-Debugging/src/02-Custom-Type-Errors/03-Functions.purs#L28

Reply:
Fixed.